src/en/clowncorps/src/eu/kanade/tachiyomi/extension/en/clowncorps/ClownCorps.kt

```diff
@@ -26,14 +26,8 @@ import java.util.Locale
 class ClownCorps : ConfigurableSource, HttpSource() {
     override val baseUrl = "https://clowncorps.net"
     override val lang = "en"
-    override val name = "Clown Corps"
-    override val supportsLatest = false
-
-    private val creator = "Joe Chouinard"
-
-    // Text from: https://clowncorps.net/about/
-    private val synopsis = "Clown Corps is a comic about crime-fighting clowns. " +
-        "It's pronounced \"core.\" Like marine corps."
+    override val name = NAME
+    override val supportsLatest = SUPPORTS_LATEST
```

Comment:
Inline `name` and `supportsLatest`. I was asking you to move the creator constant only.

Reply:
Yeah I realised that too just now. Sorry, I don't use Kotlin a lot, so I'm not yet too caught-up on the exact conventions.
I'm learning quickly, though.
My next extensions will hopefully be better from the get-go.